src/api.tsx

```diff
@@ -105,7 +105,7 @@ const getReporterStats = () =>
 const getTeamStats = (eventKey: string, team: string) =>
   get<Report[]>(`events/${eventKey}/teams/frc${team}/reports`)
 
-const getUsers = () => get<UserInfo[]>('users')
+const getUsers = () => get<(UserInfo & { isVerified: boolean })[]>('users')
```

Comment:
Why did you not just modify the `UserInfo` type?

Reply:
Because UserInfo is technically only JWT stuff

Reply:
Sure